tmp/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
sounds like it'd cause some excess io processing.... i'm guessing there's no way to just cache it first, or does it need a static file?